modules/nw-sriov-installing-operator.adoc

```diff
@@ -36,7 +36,7 @@ metadata:
 EOF
 ----
 
-. To create an OperatorGroup CR, enter the following command:
+. Create an OperatorGroup CR by entering the following command:
```

Comment:
Can you make sure that "CR" is defined above? Otherwise, we might want to spell the first one out. Eg:

". . . custom resource (CR). . ."

```suggestion
. Create an `OperatorGroup` CR by entering the following command:
```

Reply:
I didn't see that CR was defined anywhere, so I went ahead and made the suggested change.